src/components/ConcurrencyLimitsTable.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <p-table class="concurrency-limits-table__table" :columns="columns" :data="concurrencyLimits">
+  <ConcurrencyLimitsPageEmptyState v-if="empty" />
```

Comment:
Interesting to see the empty state in the orion-design table component. Usually orion-ui and nebula-ui implement them both separately. Is this a conscious choice?

Reply:
I'm open to either but since the subscription is called here it seemed more logical to add the empty state and associated logic here. I think the pattern of putting the empty state in Nebula-ui and Orion-ui came about because we were doing most api calls there, which increasingly isn't necessary.

Reply:
Yep makes sense